catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/bower.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "catalog-admin-module-registry",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "backbone-relational": "0.8.8",
+    "backbone.modelbinder": "v1.0.5",
+    "components-backbone": "1.1.0",
+    "backbone-poller": "0.2.8",
+    "components-bootstrap": "3.1.1",
+    "components-font-awesome": "4.0.3",
+    "icanhandlebarz": "atomicobject/ICanHandlebarz.js#f6ac1d1f8e1d796045563f1483eca6c4911ed7d1",
+    "bootstrap-multiselect": "v0.9.3",
+    "handlebars": "2.0.0-alpha.2",
+    "html5shiv": "3.7.2",
+    "jquery": "components/jquery#1.11.0",
+    "jquery-ui": "1.10.4",
+    "lodash": "2.4.1",
+    "marionette": "1.8.8",
+    "perfect-scrollbar": "0.4.8",
+    "q": "v1.0.1",
+    "requirejs": "2.1.14",
+    "requirejs-plugins": "v1.0.2",
+    "require-css": "0.1.5",
+    "spin.js": "1.3.3",
+    "bootswatch": "=v3.1.1",
+    "jquery-file-upload": "9.5.7",
```

Comment:
Initially we were going to but it didn't make it into the ui yet. I will remove it for now.